tests/test_mem_eff_attention.py

```diff
@@ -1581,7 +1608,7 @@ def test_decoder(
     # kv_heads = 1: multiquery
     # kv_heads = None: neither MQA nor GQA
     # kv_heads > 1: BMGHK
-    if dtype == "bf16" and compute_capability < (8, 0):
+    if dtype == "bf16" and torch.version.cuda and compute_capability < (8, 0):
         raise pytest.skip("BF16 is only supported on SM80+")
     import triton
 
```

Comment:
For this file, wonder if we can remove more `disable_on_rocm` for more tests after this PR?

Reply:
Remove the whole `test_memory_efficient_attention_full_block_masked`,  it seems this one is not used by upstream codes